unittests/allplatformstests.py

```diff
@@ -2020,7 +2020,7 @@ def test_options_listed_in_build_options(self) -> None:
 
     def test_subproject_promotion(self):
         testdir = os.path.join(self.unit_test_dir, '12 promote')
-        workdir = os.path.join(self.builddir, 'work')
+        workdir = os.path.join(self.builddir, '..' + os.path.sep + 'work')
```

Comment:
This should be `os.path.join(self.builddir, '..', 'work')`. The same for for the other usages below.

Reply:
Done